go/vt/vtctl/query.go

```diff
@@ -244,11 +246,13 @@ func commandVtGateExecuteShards(ctx context.Context, wr *wrangler.Wrangler, subF
 
 	bindVars, err := sqltypes.BuildBindVariables(*bindVariables)
 	if err != nil {
+		//lint:ignore ST1005 function name
```

Comment:
It looks like staticcheck tries to allow function names from the current package at the beginnings of error messages: https://github.com/dominikh/go-tools/blob/c51f1e51d5a3f3c686a9e8229818d34e8fe4e547/stylecheck/lint.go#L401 

Seems like there are some interesting questions here:
- Should this function log a name other than its own?
- Should this function log anything at all? What value is it adding by wrapping the error, and should it just use `vterrors.Wrap()`?

Reply:
staticcheck does not allow function names if they are one word. I changed the logging in a few places to get around this.

Reply:
Staticcheck allows (should allow) function names (regardless of the number of words); independently of that it also allows CamelCase words, assuming they're names or initialisms we didn't catch otherwise. I'd love to know about the cases in which this didn't work.

Reply:
This block of code produces an error:
```
	qr, err := session.Execute(ctx, subFlags.Arg(0), bindVars)
	if err != nil {
		return fmt.Errorf("Execute failed: %v", err)
	}
```
query.go:198:20: error strings should not be capitalized (ST1005)